cdap-api-common/src/main/java/io/cdap/cdap/api/exception/ErrorCategory.java

```diff
@@ -56,6 +58,16 @@ public String getErrorCategory() {
         errorCategory.toString() : String.format("%s-'%s'", errorCategory, subCategory);
   }
 
+  @Override
```

Comment:
Please add hashCode as well (and fix other checkstyle problems)

Reply:
done.